test/powershell/Modules/Microsoft.PowerShell.LocalAccounts/Pester.Command.Cmdlets.LocalAccounts.LocalGroup.Tests.ps1

```diff
@@ -335,13 +328,13 @@ try {
             if ($null -eq $result)
             {
                 # Force failing the test because an unexpected outcome occurred
-                $false | Should Be $true
+                $false | Should -BeTrue
```

Comment:
Can I remove the `if/else` and just `$result | Should -Not -BeNullOrEmpty` instead

Reply:
No.